hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/UpdateTargetWindowController.java

```diff
@@ -98,10 +105,14 @@ protected Class<ProxyTarget> getEntityClass() {
     @Override
     protected boolean isEntityValid(final ProxyTarget entity) {
         final String controllerId = entity.getControllerId();
-        return proxyTargetValidator.isEntityValid(entity, () -> hasControllerIdChanged(controllerId)
-                && targetManagement.getByControllerID(controllerId).isPresent());
+        return proxyTargetValidator.isEntityValid(entity,
+                () -> contextAware.runAsTenant(
+                        contextAware.getCurrentTenant(),
+                        () -> hasControllerIdChanged(controllerId)
+                                && targetManagement.getByControllerID(controllerId).isPresent()));
```

Comment:
contextAware#runAsTenant based on contextAware#getCurrentTenant. I assume this will temporary deactivate the access restrictions. However this looks confusing for me, since the affect is not recognizable

Reply:
Added comment. I think modified just 4 places where you already elevated access. Maybe we could add on all palaces. However, not sure we need to full integration with UI?